lib/mind_the_gapp_web/router.ex

```diff
@@ -11,12 +11,13 @@ defmodule MindTheGappWeb.Router do
 
   pipeline :api do
     plug :accepts, ["json"]
+    plug CORSPlug, origin: "http://localhost:8000"
```

Comment:
I have a feeling this should not be hardcoded to `localhost`. This will break when it is deployed, yeah?

Reply:
I'll refactor it to be an env variable